src/app/reports-handler.ts

```diff
@@ -102,6 +108,28 @@ export const exportLogs = (): void => {
         });
 };
 
+class RetrieveLogs {
```

Comment:
Do we really need to have a class?? we can just export the variable.

Reply:
I tried, but I couldn't get it to work, I kept getting errors. Probably me doing something wrong... What would that look like?

Reply:
in `src/app/reports-handler.ts`
```
export const retrieveLogs: IRetrieveLogs = {
    webContents: undefined
};
```

Create `IRetrieveLogs` in `src/common/api-interface.ts`
```
export interface IRetrieveLogs {
    webContents: Electron.WebContents | undefined,
}
```

Reply:
ok, yes, that works, thanks :)